usaspending_api/api_contracts/contracts/v2/search/spending_by_subaward_grouped.md

```diff
@@ -0,0 +1,291 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Spending By Subaward Grouped
+
+## POST
+
+This endpoint takes fields and award filters and returns a list containing filtered award ids, the number of subawards in each award, the total amount of obligations of all the subawards, and the award unique id
+
++ Request (application/json)
+    + Schema
+
+            {
+                "$schema": "http://json-schema.org/draft-04/schema#",
+                "type": "object"
+            }
+
+    + Attributes (object)
+        + `filters` (required, AdvancedFilterObject)
+        + `limit` (optional, number)
+            How many results are returned
+        + `page` (optional, number)
+            The page number that is currently returned.
+        + `sort`: Any field within the result can be sort (optional, enum[string])
+            The field on which to order `results` in the response.
+            + Default: `award_id`
+            + Members
+                + `award_id`
```

Comment:
Won't block over this, but if you have time would like to try and keep list like this and the response alphabetized at least in the documentation, so it is easier to find what you need.